skglm/solvers/cd_solver.py

```diff
@@ -112,6 +107,8 @@ def cd_solver_path(X, y, datafit, penalty, alphas=None,
     n_alphas = len(alphas)
 
     coefs = np.zeros((n_features, n_alphas), order='F', dtype=X.dtype)
+    if fit_intercept:
+        intercepts = np.zeros((1, n_alphas), order='F', dtype=X.dtype)
```

Comment:
there is no need to specify fortran order as the array has only one row, it's both C and F contiguous